tgui/packages/tgui-panel/ping/PingIndicator.js

```diff
@@ -19,9 +20,16 @@ export const PingIndicator = (props) => {
   ]);
   const roundtrip = ping.roundtrip ? toFixed(ping.roundtrip) : '--';
   return (
-    <div className="Ping">
+    <Button
+      lineHeight="15px"
+      width="50px"
+      className="Ping"
+      color="transparent"
+      tooltip="Ping relays"
+      tooltipPosition="bottom-start"
+      onClick={() => act('ping_relays')}>
       <Box className="Ping__indicator" backgroundColor={color} />
```

Comment:
Also I noticed that the whole thing was made slightly wider and that the text is greyed out until hovered over. Not sure if that's intentional or not but I thought I'd mention anyway.

Reply:
The light theme hover state just really has no indication so I will keep the text color as is. `textColor="label"` would be the next best I can find; but I think it looks fine as is.